guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+
+## Signing in
+
+The biggest hurdle at this stage is that most users will not be familiar with this authentication method and are not likely to have a wallet they can readily use. Ensure that there are concise explanations and tips to get users started.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info@2x.png"
+   alt-text = "Sign in screen with tips for wallets to use and general information"
+   width = 800
+   height = 411
+%}
+
+Authentication is a simple process in which the service sends a message to the wallet. The wallet then uses cryptography to generate a unique identifier from the message, which it sends back to the service. This identifier can then be used to establish an account.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-request.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-request@2x.png"
+   alt-text = "Browser window with a modal for approving web service authentication"
+   width = 800
+   height = 411
+%}
+
+Keep in mind that your service will not be able to retrieve user names, emails, balances, transactions or other information from the wallet. Only the minimal information to identify the user is communicated. If you need to communicate with users about their accounts or activity, make sure you request an email or other contact point.
+
+## Signing out
+
+Ensuring users are signed out (manually or via session time-outs) is important. The account may have payment allowances, or other problematic permissions, that could otherwise be abused by third-parties.
+
+## Making payments
+
+A payment is typically initiated by your service when the user wants to take an action. Show the most appropriate option right away based on how the user has previously interacted with their wallet (direct or bridged, [see above]({{ '#wallet-service-communication' | relative_url }})).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/payment-request.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/payment-request@2x.png"
+   alt-text = "Browser window with a modal for confirming a payment of 100 satoshi"
+   width = 800
+   height = 411
+%}
+
+Allowances allow for automatic approval of payments by wallets up to certain amounts. This reduces friction for users that frequently transact. With an allowance, the modal window in the visualization above would not be needed. Ensure your interface provides user feedback when payments are confirmed.
+
+An alternative to reducing friction for frequent payments is to let users deposit satoshi into their accounts (and later withdraw them). This puts the service into a custodial position, for the benefit of smoother payments (since they only involve database updates and not actual lightning payments).
```

Comment:
```suggestion
An alternative to reducing friction for frequent payments is to let users deposit satoshi into their accounts (and later withdraw them). This puts the service into a custodial position for the benefit of smoother payments (since they only involve database updates and not actual lightning payments).
```